zerver/actions/message_edit.py

```diff
@@ -961,6 +967,22 @@ def check_update_message(
         if (timezone_now() - message.date_sent) > datetime.timedelta(seconds=deadline_seconds):
             raise JsonableError(_("The time limit for editing this message's topic has passed"))
 
+        if check_time_limit_constraints and propagate_mode == "change_all":
+            message_query = Q(
```

Comment:
Let's call this something like `disallowed_messages_query`

Reply:
Udpated this to `allowed_messages_query` since I have changed the logic now.